src/training/ui_hacks.rs

```diff
@@ -106,6 +106,20 @@ const BG_LEFT_OFF_BLACK_COLOR: ResColor = ResColor {
     a: 0,
 };
 
+const BG_LEFT_SELECTED_BLACK_COLOR: ResColor = ResColor {
+    r: 80,
+    g: 0,
+    b: 0,
+    a: 0,
+};
+
+const BG_LEFT_SELECTED_WHITE_COLOR: ResColor = ResColor {
+    r: 118,
+    g: 0,
+    b: 0,
+    a: 255,
+};
+
 macro_rules! menu_text_name_fmt {
```

Comment:
Is it possible to use this formatting mechanism for your slider labels as well? This way we don't have to copy pasta the literals around and potentially get them wrong between build() and draw().